### PR TITLE
Change tab URL to not pre-select based on current value

### PR DESCRIPTION
**Pull request by aschempp**

### CODE REVIEW

File: src/Picker/ArticlePickerProvider.php [hunk 1]
```diff
@@ -82,7 +82,7 @@ protected function getLinkClass()
     /**
      * {@inheritdoc}
      */
-    protected function getRouteParameters(PickerConfig $config)
+    protected function getRouteParameters(PickerConfig $config = null)
```
Comment (leofeyer) [inline src/Picker/ArticlePickerProvider.php]: It seems that you are not using `$config` in any of the child classes?

Comment (aschempp) [inline src/Picker/ArticlePickerProvider.php]: Because the core pickers are not picking from subtables. News, Event and FAQ pickers do.

Comment (leofeyer) [inline src/Picker/ArticlePickerProvider.php]: Yes, but `null` is never passed as argument, is it? We could leave it as it was.

Comment (aschempp) [inline src/Picker/ArticlePickerProvider.php]: If that was true, then the URL would always link to the selected value 😉 
see https://github.com/contao/core-bundle/pull/984/files#diff-4e7f624198eef84ce3f501384ca6dcc8R135

File: src/Picker/AbstractPickerProvider.php [hunk 2]
```diff
@@ -122,6 +122,15 @@ protected function getUser()
     }
 
     /**
+     * Returns the routing parameters for the backend picker.
```
Comment (xchs) [inline src/Picker/AbstractPickerProvider.php]: backend -> back end

**Pull request merged**